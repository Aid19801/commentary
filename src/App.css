.App {
  text-align: center;

  display: flex;
  flex-direction: column;
  
  box-sizing: border-box;
  background: url('images/football.jpg')  50% 0 fixed;
  background-size: cover;
  height: 100vh;
  margin: 0 auto;
  width: 100%;
  position: relative;
}

.App-logo {
  animation: App-logo-spin infinite 20s linear;
  height: 40vmin;
}

.App-header {
  background-color: #282c34;
  min-height: 100vh;
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;
  font-size: calc(10px + 2vmin);
  color: white;
}

.App-link {
  color: #61dafb;
}

@keyframes App-logo-spin {
  from {
    transform: rotate(0deg);
  }
  to {
    transform: rotate(360deg);
  }
}
