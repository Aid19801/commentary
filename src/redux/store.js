import { createStore, applyMiddleware } from 'redux';

import createSagaMiddleware from 'redux-saga';
import { all } from 'redux-saga/effects'
import reducers from './updates-reducer';

import { watcherGetUpdates } from './sagas';


const sagaMiddleware = createSagaMiddleware();

const store = createStore(reducers, applyMiddleware(sagaMiddleware));

function* rootSaga() {
    yield all([
        watcherGetUpdates(),
    ])
}

sagaMiddleware.run(rootSaga);

export default store;