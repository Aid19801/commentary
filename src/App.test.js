import React from 'react';
import ReactDOM from 'react-dom';
import App from './App';
import { Provider } from 'react-redux';
import store from './redux/store';

import renderer from 'react-test-renderer';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(
    <Provider store={store}>
      <App />
    </Provider>, div);
  ReactDOM.unmountComponentAtNode(div);
});

it('renders correctly', () => {
  const tree = renderer
    .create(
    <Provider store={store}>
      <App />
    </Provider>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
