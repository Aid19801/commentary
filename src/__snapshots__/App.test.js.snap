// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="App"
>
  <div
    className="homepage"
  >
    <div
      className="updates-section"
    >
      <div
        className="title"
      >
        <h4
          className="title-text"
        >
          Live Commentary
        </h4>
      </div>
      <div
        className="updates-container"
      />
    </div>
    <div
      className="key-moments-container"
    >
      <div
        className="title"
      >
        <h4
          className="title-text"
        >
          Key Moments
        </h4>
      </div>
      <div
        className="key-moments-section"
      />
    </div>
  </div>
</div>
`;
